src/web/src/app/space/service.ts

```diff
@@ -25,15 +28,17 @@ export class SpaceService {
 	constructor(
 		private http: HttpClient,
 		private requestWatcher: RequestWatcherService,
+		private apiSpaceService: ApiSpaceService,
```

Comment:
Давай покороче попробуем
```suggestion
		private api: ApiSpaceService,
```

Reply:
Поправил везде.